pallets/nfts/src/lib.rs

```diff
@@ -1371,6 +1529,55 @@ pub mod pallet {
 			Self::do_clear_all_transfer_approvals(maybe_check_origin, collection, item)
 		}
 
+		/// Cancel all the collection approvals.
+		///
+		/// Origin must be Signed.
+		///
+		/// Arguments:
+		/// - `collection`: The collection whose approvals will be cleared.
+		/// - `witness_approvals`: Information on the collection approvals cleared. This must be
```

Comment:
What information, why not simplify and be specific that this is the amount of collection approvals that want to be removed?